Cargo.toml

```diff
@@ -55,12 +55,8 @@ protobuf = { version = "2", features = ["with-bytes"] }
 clap = { version = "4.0.32", features = ["derive"] }
 semver = "1.0"
 cw-semver = { version = "1.0" }
-cw-orch = { version = "0.11" }
+cw-orch = { package="cw-orch-publish-test", version = "0.11" }
```

Comment:
This is neede because we mutate the state of a daemon after instantiation. This is the following cw-orch branch : https://github.com/AbstractSDK/cw-orchestrator/pull/137